main.c

```diff
@@ -484,6 +484,8 @@ static void compute_file_name(char **strp,
       int dir_fd;
       int ret = 0;
       uid_t euid = geteuid();
+      const char *sock_group_name = cfg_get_string(NULL, "sock_group", NULL);
```

Comment:
I can see it being potentially useful to allow this to also be the gid (eg: we have a consistent gid for a role but deploy across systems where that group has a different name in the local groups file), but we can tackle that in the future.